src/htsmsg.c

```diff
@@ -303,6 +303,7 @@ htsmsg_destroy(htsmsg_t *msg)
   memoryinfo_free(&htsmsg_memoryinfo, sizeof(htsmsg_t));
 #endif
   free(msg);
+  msg = NULL;
```

Comment:
This is the last instruction in the function, setting it to NULL simply does nothing.